site/src/components/table.md

```diff
@@ -93,6 +94,231 @@ For cases in which a table has many columns, the `table-wrapper` class is availa
 
 ## Modifiers
 
+_`table-simple`_ - simple table style only shows horizontal cell borders and no table border.
```

Comment:
Thinking that this page has gotten extremely long. Just floating the idea, but perhaps we could just abbreviate or remove the code blocks for examples?

Reply:
I don't like how long the page became either, but I couldn't find out the best solution to shorten it. Removing all examples won't work, because I think some cases should be presented such as stacking on mobile, or how the complex table looks since it's pretty different from the default. And at the same time having examples for a few cases and not all feel weird to me. I'll try to shorten the code examples, showing only 2 rows in the table instead of 3 might work.

Reply:
What about just removing the code blocks, not the examples?

Reply:
Updated

Reply:
If we want to shorten the page, I'm wondering if the first example can be combined with the second example?

Reply:
I think it's fine the way it is right now after the latest update - having two code examples is much better than 9 as it used to be :)

Reply:
I agree that the second table-wrapper example code seems like it could be removed.

Reply:
Updated